README.md

```diff
@@ -367,6 +367,26 @@ if (typeof value === 'symbol') {
 
 ```
 
+## `.labrc` file
+
+**lab** supports a `.labrc` configuration file for centralizing lab settings.  
+The `.labrc` file can be located in the current working directory or in the
+users home directory.  The `.labrc` file needs to be able to be required by
```

Comment:
The second sentence might start better as "The .labrc file must be requirable by"